GridKit/LinearAlgebra/DenseMatrix/DenseMatrix.hpp

```diff
@@ -52,14 +52,14 @@ namespace GridKit
     /**
      * @brief DenseMatrix constructor
      *
-     * @tparam ScalarT
+     * @tparam RealT
      * @tparam IdxT
```

Comment:
Perhaps explain what these data types are used for:
```suggestion
     * @tparam RealT - Real type for Jacobian entries
    * @tparam IdxT - Integer data type for matrix indices
```

Reply:
Addressed in [d6eaba1](https://github.com/ORNL/GridKit/pull/305/commits/d6eaba1f269405adfe00b62a08ed2f3fa9ea2721)